### PR TITLE
Meeting change - 2016-10-01

### PR DESCRIPTION
**Pull request by PhilConnor**
**Implemented changes as discussed in meeting 2016-10-01**
- Commenting
- Refactoring
- Removed 30th write mode.
- Moved coord conversion logic to a new CoordConverter class.
- Remove problematic dependencies to global configs.
- Moved some Cursor logic to Editor to break dependencies to data that Cursor should not have awareness.
- Added mouse drag of the cursor.
- Element value is now a single char.
- Some other minor changes.

### CODE REVIEW

**Comment by PhilConnor** [comment]
@guilbaultj Waiting for your approval.

**Review by guilbaultj — APPROVED** [review]
Approved!

**Pull request merged**
